pyproject.toml

```diff
@@ -268,16 +292,78 @@ optional = true
 [tool.poetry.dependencies.sentencepiece]
 version = "~0.1.96"
 optional = true
-extras = [ "sentencepiece",]
+extras = ["sentencepiece"]
 
 [tool.poetry.dependencies.jieba]
 version = ">=0.39, <0.43"
 optional = true
 
 [tool.poetry.dependencies.pymongo]
 version = ">=3.8,<3.11"
-extras = [ "tls", "srv",]
+extras = ["tls", "srv"]
 
 [tool.poetry.dev-dependencies.pytest-sanic]
 git = "https://github.com/RasaHQ/pytest-sanic"
 branch = "fix_signal_issue"
+
+[tool.mypy]
```

Comment:
I took the opportunity to move the configuration of mypy and pytest into pyproject.toml. It reduces the nb of config files we have :D

Reply:
thanks a lot 🔥